components/phpunit_bridge.rst

```diff
@@ -232,6 +232,51 @@ test::
 
 And that's all!
 
+.. caution::
+
+    The ``@group time-sensitive`` annotation is equivalent to ``ClockMock::register(MyTest::class)``,
+    so if you want to get a time based function mocked into one of the source code you will need to
+    add it explicitly using ``ClockMock::register(MyClass::class)``. The ``ClockMock::register`` method
+    only create a mock of the time based functions into the same namespace as your class. So when using
+    ``time()`` you will use the mock instead of the default one.
+
+
+.. code-block:: php
```

Comment:
Can be replaced by `::` at the end of line 241 instead of the dot.

Reply:
Indeed, that's what I meant. Fon't bother you can push and let our host hook build the demo, its link will be accessible in the PR checks.

Reply:
Unfortunately, it isn't deployed on Platform.sh as specified here https://symfony.com/doc/current/contributing/documentation/overview.html#review-your-changes while branch 2.8 is in a maintained branch.